vllm/entrypoints/openai/api_server.py

```diff
@@ -124,7 +124,7 @@ async def create_completion(raw_request: Request):
 
     model_name = request.model
     request_id = f"cmpl-{random_uuid()}"
-    prompt = request.prompt
+    prompt = request.prompt if type(request.prompt) != list else ' '.join(request.prompt)
```

Comment:
Let's change it to something like this:
```suggestion
    if isinstance(request.prompt, list):
        assert len(request.prompt) == 1
        prompt = request.prompt[0]
    else:
        prompt = request.prompt
```

Reply:
Thanks for this suggestion. I did misunderstand it. Let's do it in this way.